src/BuildingRegistry.Migrator.Building/Infrastructure/SqlStreamsTable.cs

```diff
@@ -6,39 +6,73 @@ namespace BuildingRegistry.Migrator.Building.Infrastructure
     using Dapper;
     using Microsoft.Data.SqlClient;
 
-    public class SqlStreamsTable
+    public class SqlBigStreamsTable : SqlStreamsTable
     {
-        private readonly string _connectionString;
-        private readonly int _pageSize;
+        public SqlBigStreamsTable(string connectionString, int pageSize = 10)
```

Comment:
Should we do this one by one? Otherwise bigbuildings will also be in parallel and will interfere with the small buildings.